### PR TITLE
add new pools on mode

### PR DESCRIPTION
**Pull request by 0xKMG**

### CODE REVIEW

**Comment by llamatester** [comment]
Error while running adapter at projects/airpuff: 
        
 
  
    
    
    Error: Llama RPC error! methos: undefined 
    - host: https://1rpc.io/eth error: execution reverted
    - host: https://cloudflare-eth.com error: execution reverted
    - host: https://ethereum-rpc.publicnode.com error: execution reverted
    - host: https://rpc.blocknative.com/boost error: execution reverted
    - host: https://rpc.mevblocker.io error: execution reverted
    - host: https://api.tatum.io/v3/blockchain/node/ethereum-mainnet error: execution reverted
    - host: https://rpc.flashbots.net error: execution reverted

**Comment by 0xKMG** [comment]
Anything I have to do to fix the rpc error? thanks

File: projects/airpuff/index.js [hunk 1]
```diff
@@ -181,7 +207,12 @@ module.exports = {
       }
 
       const mswETH = {
-        valut: "0x7c505E03460aEF7FE88e218CC5fcEeCCcA4C4394",
+        vault: "0x7c505E03460aEF7FE88e218CC5fcEeCCcA4C4394",
+        reStakingToken: "0x32bd822d615A3658A68b6fDD30c2fcb2C996D678",
+      };
+
+      const mswETH1x = {
+        vault: "0x1100195fbdA2f22AA6f394E6C65f168779Fe572c",
```
Comment (g1nt0ki) [inline projects/airpuff/index.js:215]: I dont think it is rpc issue, I dont see `getAllEigeinPieCycleDepositAmounts()` here: https://etherscan.io/address/0x1100195fbdA2f22AA6f394E6C65f168779Fe572c#readProxyContract

**Comment by llamatester** [comment]
The adapter at projects/airpuff exports TVL: 
        
 
 
    mode                      3.41 k
    arbitrum                  7.40 M
    ethereum                  3.48 M
    
    total                    10.88 M

**Comment by 0xKMG** [comment]
Thanks for the notice, fixed now.

**Pull request merged**